contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,108 @@
+
+## What are the skills a growth engineer needs?
+
+Growth engineers are responsible for three key tasks:
+
+1. Creating ideas and strategies for growth.
+2. Implementing those ideas and strategies.
+3. Measuring and evaluating those implementations.
+
+To accomplish these tasks, they need a combination of engineering, business, product, and data skills.
+
+First and most obvious, is **engineering skills**. Growth engineers write code to implement their ideas and experiments. If they couldn’t code, they wouldn’t be an engineer, and that is half the name. They must be comfortable across the stack, use new tools like external APIs, and work we unfamiliar tech if necessary.
+
+Second, the other part of creating and implementing ideas is **product and design knowledge**. Because a growth team works across the stack and teams, they are often "lone wolves." They need the ability to fend for themselves in product and design decisions. They don’t have to be Da Vinci, but product and design knowledge helps them create successful features.
+
+Third, to create growth ideas based on solid strategies, growth engineers must have an **understanding of business**. This includes areas like marketing and finance. This is because growth engineers work to improve core business metrics. If they don’t have an understanding of business fundamentals, they can’t be effective at improving those metrics. Growth engineers also regularly work with non-technical teams, and so they need this knowledge to collaborate well with them.
+
+Last is **data literacy**. Since growth engineers do a lot of experiments, they must understand how those experiments perform. They should know what factors are relevant, how to calculate if something is significant, and what data to care about in the first place. All of this requires at least a basic knowledge of data and statistics.
+
+![Growth engineers' power](../images/blog/what-is-a-growth-engineer/thanos.jpg)
+
+If this sounds like a unicorn, you aren’t wrong. This is again why we find former technical founders as a good fit for this role. They have the business and product knowledge to develop growth ideas and strategies, the practical technical knowledge to implement them, as well as a familiarity with the data a product produces.  
+
+## What do growth engineers actually work on?
+
+So what do growth engineers use their skills to do? We’ve already covered that they work to increase signups, subscriptions, and revenue by working across the product, but what are some examples of what this looks like?
+
+- Developing a growth strategy, experiment framework, and measurement for both.
+- Doing SEO optimization on the marketing site. Building landing pages. Optimizing the signup funnels.
+- Running A/B tests on specific parts of the signup and activation funnel. Trying to improve the conversion rate for both.
+- Setting up in-app and push notification messaging systems. Building tools and systems for better product onboarding.
+
```

Comment:
In my experience they usually work on 2 different types of things:
1. Grow the target audience of the product / Working on a large projects that will unlock growth to a specific set of users that were unable to use the app before. A good example is Uber adding "pay with Cash" feature in India, since most people there dont use credit cards
2. Moving Activation/Acuqisition metrics. This usually involves large projects to help users understand and use the product better in their first few days (i.e. reach their "aha" moment sooner). It also involves tons of experiments, small and large, to try and move these metrics

Reply:
Got it added new markets and adjusted signup and activation bullet.